Server/GameEngine.cpp

```diff
@@ -87,7 +110,9 @@ void GameEngine::movePlayers(vector<PlayerInputs> & playerInputs) {
 	// Move all players
 	for (int i = 0; i < gameState.players.size(); i++) {
 		// TODO: prevent two players from moving to the same spot
-
+		if (gameState.players[i] == nullptr) {
+			std::cerr << "Whoa" << std::endl;
```

Comment:
Nice lol. Would this mean a player suddenly disappeared?

Reply:
This was left over from a bug I had, but it has been fixed now.